android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -132,7 +157,51 @@ protected void triggerLayoutInflation() {
 
         TextView assetTitleText = findViewById(R.id.asset_title_text);
         assetTitleText.setText(mAssetName);
-        if (!mAssetLogo.isEmpty()) {
+        mBtnBuy = findViewById(R.id.btn_buy);
+        Button btnSend = findViewById(R.id.btn_send);
+        mBtnSwap = findViewById(R.id.btn_swap);
+
+        if (mCoinMarket) {
```

Comment:
`mCoinMarket` boolean should be renamed to `mIsMarketCoin`.

Reply:
I this is a standard practice and make more sense. [Boolean variables should be prefixed with ‘is’](https://www.comp.nus.edu.sg/~cs2103/AY1516S2/contents/coding-standards-java.html)

Reply:
I'm changing also its related Intent constant `COIN_MARKET` -> `IS_MARKET_COIN`, I think it makes more sense.

Reply:
> This is a standard practice and make more sense. [Boolean variables should be prefixed with ‘is’](https://www.comp.nus.edu.sg/~cs2103/AY1516S2/contents/coding-standards-java.html)

I have no problem adopting the standard of the Singapore University you linked, really.
But there are other influential sources though, a simple search on StackOverflow will show you that not for everybody makes more sense.
Have a look here -> https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter
Also JetBrains (and they _know_ how to do things 😁) decided to adopt booleans without verbs. 
This is just a random example that accepts both forms.
https://github.com/JetBrains/intellij-community/blob/master/plugins/lombok/testData/after/getter/GetterBoolean.java

I don't have a strong opinion, but in my experience over the years I have a mild preference for the shorter form without verb, as it's easier to obtain its related getter: `foo` easily becomes `isFoo()`.

Sometimes what we think "it makes more sense" might not be true for all.

Reply:
I see `isBar`, `hasFoo`, and `isFoo`. The example is [auto generated](https://github.com/JetBrains/intellij-community/blob/master/plugins/lombok/testData/after/getter/GetterBoolean.java), though another common one [isEmpty](https://docs.oracle.com/javase/6/docs/api/java/util/Map.html#isEmpty()). It may not make sense to few (we can always find some), would leave it upto @SergeyZhukovsky to choose a convention.

Reply:
I'm more than happy to change it into what you find more readable for now. It's not a big deal to me. 

Just wanted to share some food for thoughts.
If we want to model a code style for the project count me in, I would love to give a hand.

Reply:
I cannot discuss on the Android dev channel something that you suggested, so no, I won't!
This is not about project convention, as currently there's a good mix between the two approaches.

<img width="1173" alt="Cursor" src="https://github.com/brave/brave-core/assets/3308503/2d683a54-96f1-4700-beeb-ebd5033457a7">

This is about collaboration and cooperation.
A collaboration I'm not seeing I'm afraid :-/

Reply:
There are some different variable and we likely missed those in a review, though I see many examples of `has`, `should`, `is` etc in the above pic. Feel free to discuss what you think is something we should update.